ros_control.rosinstall

```diff
@@ -0,0 +1,4 @@
+- git: {local-name: ros_control, uri: 'https://github.com/ros-controls/ros_control.git', version: noetic-devel}
+- git: {local-name: realtime_tools, uri: 'https://github.com/ros-controls/realtime_tools.git', version: noetic-devel}
```

Comment:
You forgot to add `ros_controllers`. Additionally, the `.travis.yml` does not define UPSTREAM_WORKSPACE as global and keep only the first two jobs as all the other ros_controls' packages do (https://github.com/ros-controls/ros_control/blob/fc5a5d7f4df57de201ae5d97df42f2d754faf1a8/.travis.yml#L14-L19). 

The CI tests work fine after these 2 updates (I tested locally with industrial_ci. I didn't open a Pull Request to not override this one)

I also would like to comment that, if you prefer to define the .rosinstall config file in a single place, it must be placed here (as you did) instead of ros_control because `ros_controllers` is at the tip of the dependency tree.

Reply:
My memories of how thing works are all fuzzy now :)

But I do know that why it was used here the way it was (and probably still should be) is to provide a full source-build as far as ros_control is concerned. This should flag up build issues between the two more tightly coupled repos (ros_control, ros_controllers).

Reply:
I got your point. I'm sorry for the back-and-forth moves, I was not used to industrial_ci and I didn't have much time to dig deeper into this problem.

Now I'm confident the last updates I proposed will solve the problem (at least I hope we move a step forward).